### PR TITLE
UI enhancements

### LINKED ISSUE #176: Settings > Voting policy dropdown not showing correct follow up fields
# Current State
When I load the manage members group, I see that the selected voting policy is Number of Votes. However, the wrong follow up filed is shown (e.g. percentage):
Image

# Desired State
The correct follow up field should show for number of votes:
Image

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Resolves #176 #164

### CODE REVIEW

**Comment by Megha-Dev-19** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@Megha-Dev-19 Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!


[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/178&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/178&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Megha-Dev-19)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Megha-Dev-19)

For maintainers:
- Score pull requests that participate in the Race of Sloths and receive a reward
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by petersalomonsen** [comment]
I don't see a test that goes through the reproduction steps in https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/issues/164

Shouldn't we have that?

**Comment by petersalomonsen** [comment]
I also miss a test reproducing this: https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/issues/176

File: playwright-tests/tests/payments/create-payment-request.spec.js
Comment (petersalomonsen) [inline playwright-tests/tests/payments/create-payment-request.spec.js:306]: Are these changes in the test related to the referenced issues in the PR? I don't see that they are connected to voting policy. Should be described in the PR, so that it is possible to understand what has been changed here.

File: playwright-tests/tests/stake-delegation/stake-delegation.spec.js
Comment (petersalomonsen) [inline playwright-tests/tests/stake-delegation/stake-delegation.spec.js:988]: Also here I am wondering what issue this is related to. Is it testing a bug fix that for a bug that is not registered?

File: playwright-tests/tests/payments/create-payment-request.spec.js
Comment (Megha-Dev-19) [reply playwright-tests/tests/payments/create-payment-request.spec.js:306]: so I made [this](https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/pull/178/files#diff-fee2f2d99d350199c36da8967ec42443e2093086ff3912b0e449210646c7c28dR60
) change, which resets the create payment form to default on closure, that's why needed to fix this test.

File: playwright-tests/tests/stake-delegation/stake-delegation.spec.js
Comment (Megha-Dev-19) [reply playwright-tests/tests/stake-delegation/stake-delegation.spec.js:988]: it's because of flakiness, the `disabled` class was not visible, so I added this to make sure the dropdown is visible first and then it checks for` disabled` class

**Review by petersalomonsen — APPROVED** [review]

**Comment by Megha-Dev-19** [comment]
Note: Because of time crunch we will merge this PR w/o tests but I will create another PR for tests and link it here

**Comment by petersalomonsen** [comment]
@race-of-sloths score 5

**Pull request merged**
